mysql-persistence/src/test/java/com/netflix/conductor/mysql/dao/MySQLQueueDAOTest.java

```diff
@@ -46,37 +44,30 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-@ContextConfiguration(classes = {TestObjectMapperConfiguration.class})
+@ContextConfiguration(classes = {TestObjectMapperConfiguration.class, MySQLConfiguration.class, FlywayAutoConfiguration.class})
 @RunWith(SpringRunner.class)
+@SpringBootTest
 public class MySQLQueueDAOTest {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MySQLQueueDAOTest.class);
 
-    private MySQLDAOTestUtil testUtil;
+    @Autowired
     private MySQLQueueDAO queueDAO;
 
     @Autowired
     private ObjectMapper objectMapper;
 
-    @Rule
-    public TestName name = new TestName();
-
-    @Rule
-    public ExpectedException expected = ExpectedException.none();
+    @Autowired
+    private HikariDataSource dataSource;
```

Comment:
nit: There is no need to wire in a specific type of DataSource here, this could just be `DataSource`.